### PR TITLE
Feature k8s

### PR DESCRIPTION
**Pull request by lowkent2me**
Add helm chart for deploy app omegabank-bank. Includes:
- service
- deployment
- ingress
- service account

Application available under http://api.omega.smodata.net/bank

### CODE REVIEW

File: k8s/bank/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            - containerPort: {{ .Values.deployment.template.specNginx.ports.containerPort }}
+              name: {{ .Values.deployment.template.specNginx.ports.name }}
+              protocol: {{ .Values.deployment.template.specNginx.ports.protocol }}
+          volumeMounts:
+            - name: {{ .Values.volumeMounts.name }}
+              mountPath: {{ .Values.volumeMounts.mountPath }}
+        - name: {{ include "bank.fullname" . }}
+          image: {{ .Values.image.repository }}:{{ .Values.image.tag | default "latest" }}
+          env:
+          - name: PG_DATABASE
```
Comment (ggramal) [inline k8s/bank/templates/deployment.yaml:38]: lets not hardcode this values. This values must be settable

Comment (lowkent2me) [reply k8s/bank/templates/deployment.yaml]: All hardcode values ​​are described in value file

File: k8s/bank/values.yaml [hunk 2]
```diff
@@ -0,0 +1,126 @@
+# Default values for bank.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 2
```
[the remark this answers is not in the document]
Comment (lowkent2me) [reply k8s/bank/values.yaml]: Files with variables are divided into their corresponding environments

**Review by ggramal — APPROVED** [review]

File: k8s/bank/values-prd.yaml [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+    maxSurge: 2
+  template:
+    specNginx:
+            name: nginx
+            image: nginx:1.25.2
+            ports:
+                containerPort: 80
+                name: http
+                protocol: TCP
+  env:
```
Comment (ggramal) [inline k8s/bank/values-prd.yaml:122]: envs:
- name: sOME_NAME
  value: somevalue
- name: some_secret_name
   valueFrom:
       secretKeyRef:
            key: somekey
             name: some-secret-name

**Review by ggramal — APPROVED** [review]

**Pull request merged**